### DISCUSSION TITLE
Short-Cut-List behind Questionmark-Button

### DISCUSSION THREAD

**Discussion by Paul8043** [bernhard-42/vscode-ocp-cad-viewer#23] [Q&A]
I have tried to look in every corner of the OCP-CAD-Viewer and found a very valuable short-cut-list.
I run Build123d under Windows and there the Windows-Key (in your list called "Meta") interferes with the Windows-Start-Menu. So, whenever I try to hide an element (which works perfectly), my screen gets overcrowded with the Windows-Start-Menu. It stays there until the Windows-Key is pressed a second time.

![ShortCuts](https://github.com/bernhard-42/vscode-ocp-cad-viewer/assets/127483918/307b79c8-ee6f-4281-b6be-847b0a67cc46)

The yellow marked entries show this behavior. Is there a chance to change this my replacing the Meta-Modifier-Key completely or by adding further alternatives.

I have found some information on Wikipedia. Not sure what would work over all 3 platforms.
[Table-of-Keyboard-Shortcuts](https://en.wikipedia.org/wiki/Table_of_keyboard_shortcuts)

**Comment by bernhard-42** [comment]
hmm, I missed that (as noted earlier, I don't use Windows).

I think I need to map it to Ctrl on Windows (then it is in line with Ctrl-C / Cmd-C)

Could you maybe open issue for this so it doesn't get lost? 
thanks

**Reply by Paul8043** [reply]
Copied over, done.

**Reply by bernhard-42** [reply]
Thanks

**Comment by bernhard-42** [comment]
btw. the term meta comes from browser DOM (https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent#browser_compatibility)
